lib/rules/jsx-indent.js

```diff
@@ -168,18 +168,20 @@ module.exports = {
      * @return {Number} Indent
      */
     function getNodeIndent(node, byLastLine, excludeCommas) {
-      byLastLine = byLastLine || false;
-      excludeCommas = excludeCommas || false;
+      let finalLine = byLastLine;
+      let noCommas = excludeCommas;
+      finalLine = finalLine || false;
+      noCommas = noCommas || false;
```

Comment:
```suggestion
      const finalLine = !!byLastLine;
      const noCommas = !!excludeCommas;
```
altho i'm not sure either of these is needed at all, since both variables are used in a truthiness check.

Reply:
Therefore, should I make the changes that you suggested or leave as is?

Reply:
I think maybe just remove the variables entirely